.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 <!--
 Please make sure to read the Pull Request Guidelines:
-https://github.com/vuejs/vue/blob/dev/.github/CONTRIBUTING.md#pull-request-guidelines
+https://github.com/vuejs/vue/blob/dev/CONTRIBUTING.md#pull-request-guidelines
```

Comment:
This line is using the url for the next branch files, no need to touch it. Also it's usually a good idea to keep a PR focused on one thing only.